orchagent/pfcwdorch.cpp

```diff
@@ -754,6 +755,151 @@ bool PfcWdSwOrch<DropHandler, ForwardHandler>::stopWdOnPort(const Port& port)
     return true;
 }
 
+template <typename DropHandler, typename ForwardHandler>
+bool PfcWdSwOrch<DropHandler, ForwardHandler>::continueWdOnPort(const Port& port)
+{
+    SWSS_LOG_ENTER();
+
+    sai_object_id_t queueId = SAI_NULL_OBJECT_ID;
+
+    for (uint8_t i = 0; i < port.m_queue_ids.size(); i++)
+    {
+        queueId = port.m_queue_ids[i];
+
+        auto pos = m_pfcwdconfigs.find(queueId);
+        if (pos == m_pfcwdconfigs.end()) {
+            continue;
+        }
+
+        SWSS_LOG_INFO("Starting Wd on port %s", port.m_alias.c_str());
+        return PfcWdOrch<DropHandler, ForwardHandler>::createEntry(port.m_alias, pos->second);
```

Comment:
The `PFC_WD_DETECTION_TIME` and `PFC_WD_RESTORATION_TIME` are divided by 1000 when saving into `m_pfcwdconfigs`. Should multi by 1000 here?